packages/template-ui/src/components/FilterDropDown.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <b-dropdown
+    ref="dropdown"
+    class="mr-2"
+    :text="filter.prettyName"
+    :variant="filter.variant"
+    :boundary="boundary"
+  >
+    <div class="d-flex filter-dropdown flex-column">
+      <b-dropdown-header>
+        {{ filter.name }}
+        <b-button-close @click="closeFilter()" />
+      </b-dropdown-header>
+      <b-dropdown-divider />
+      <div class="filter-content flex-fill">
+        <b-btn
+          v-for="option in filter.options"
+          :key="option"
+          pill
+          class="mx-1 my-1"
+          :variant="buttonVariant(filter, option)"
+          @click="onOptionClick({ filter, option, checked: !isSelected(filter, option) })"
+        >
+          {{ option }}
+        </b-btn>
+      </div>
+      <b-dropdown-divider />
+      <b-dropdown-form class="flex-shrink-0">
+        <b-button variant="link" @click="clearFilter({ filter })">
+          Clear
+        </b-button>
+      </b-dropdown-form>
+    </div>
+  </b-dropdown>
+</template>
+
+<script>
+  import { mapGetters, mapMutations } from 'vuex';
+
+  export default {
+    name: 'FilterDropDown',
+    props: {
+      filter: Object,
+    },
+    computed: {
+      ...mapGetters({
+        isSelected: 'filters/isSelected',
+      }),
+      boundary() {
+        return document.body;
+      },
+    },
+    methods: {
+      ...mapMutations({
+        onOptionClick: 'filters/setFilterQuery',
+        clearFilter: 'filters/clearFilterQuery',
+      }),
+      buttonVariant(filter, option) {
+        return this.isSelected(filter, option) ? 'primary' : 'secondary';
+      },
+      closeFilter() {
+        this.$refs.dropdown.hide(true);
+      },
+    },
+  };
+</script>
+
+<style lang="scss" scoped>
+  @import '@/styles.scss';
+
+  .filter-dropdown {
+    max-width: 720px;
+    max-height: 480px;
+    min-width: 380px;
+    min-height: 185px;
```

Comment:
The dropdown size is all hardcoded! Can we agree in not having hardcoded values anymore? This is also not responsive. I think it should fit the 100% width in mobile. And for other breakpoints it may be a % of the breakpoint width.

About the height, why limit it?

Reply:
These are the values that are in the spec. I don't know how to handle these specs, if I should try to follow as much as possible, or if we can just adapt the spacing to existing components.

I've just updated to follow your suggestion, just remove the min-height and just limit the width using responsive breakpoints.

I've left the `max-height` limit because in other case we won't see the vertical scrollbar when there are a lot of tags, just the dropdown get bigger and can get the full height.

Reply:
Yes, I understand you. We as engineers are responsible for the technical part, so we should advice Joana if something isn't specified following good practices. Please always try to reach Joana with questions or suggestions for better implementations. I do, and she hears. She is always open to suggestions.

Also there is a bit of interpretation any time you read a spec. For instance:
- We use REM units, not pixels (Bootstrap does). But you can assume that 1rem == 16px, so you can do the conversion yourself.
- For the height maybe she meant to fix N number of rows, then start scrolling? (ask her) If so we can do the calc in the SCSS, which is better for maintenance. If the button size changes in the future the hardcoded height will break.

In the worst case she may say "I really want this box to be this min and max size, regardless of the responsiveness", in which case we should at least move  the values to the styles file, so they are documented and overrides can override them.